app/models/agents/website_agent.rb

```diff
@@ -106,24 +106,41 @@ class WebsiteAgent < Agent
 
       Set `http_success_codes` to an array of status codes (e.g., `[404, 422]`) to treat HTTP response codes beyond 200 as successes.
 
+      The `template` option is the way to format extracted data, and add meta information to each resulted payload.  Its value must be a hash, whose key-value pairs are interpolated after extraction and added to each event payload.  e.g.:
```

Comment:
Maybe note that the added data is used in the `on_change` detection? "Add meta information to each resulted payload" made me believe the added data is added after the detection.